diff --git a/source/main.cpp b/source/main.cpp
index 842c971..d8ea811 100644
--- a/source/main.cpp
+++ b/source/main.cpp
@@ -8,6 +8,7 @@
 #include "current_cfw.hpp"
 #include "fs.hpp"
 #include "main_frame.hpp"
+#include "ryazhenka_config.hpp"
 #include "warning_page.hpp"
 
 namespace i18n = brls::i18n;
@@ -29,7 +30,7 @@ int main(int argc, char* argv[])
     if (languageFile.find("language") != languageFile.end())
         i18n::loadTranslations(languageFile["language"]);
     else
-        i18n::loadTranslations();
+        i18n::loadTranslations(std::string(ryazhenka::kDefaultLanguage));
 
         //appletInitializeGamePlayRecording();
 
