diff --git a/Makefile b/Makefile
index 7ac3ff0..8d56454 100644
--- a/Makefile
+++ b/Makefile
@@ -20,10 +20,10 @@ SOURCES		:=	source
 RESOURCES	:=	resources
 DATA		:=	data
 INCLUDES	:=	include /lib/borealis/library/include/borealis/extern/nlohmann
-APP_TITLE	:=	All-in-One Switch Updater
-APP_AUTHOR	:=	HamletDuFromage
-APP_VERSION :=	2.23.3
-TARGET		:=	$(notdir $(CURDIR))
+APP_TITLE	:=	Ryazhenka Updater
+APP_AUTHOR	:=	Dimasick-git
+APP_VERSION :=	2.23.4
+TARGET		:=	ryazhenka-updater
 
 ROMFS				:=	resources
 BOREALIS_PATH		:=	lib/borealis
@@ -161,7 +161,7 @@ $(ROMFS):
 	@[ -d $@ ] || mkdir -p $@
 	@cp -rf $(CURDIR)/$(ROMFS)/i18n/zh-CN/. $(CURDIR)/$(ROMFS)/i18n/zh-Hans/
 	@cp -rf $(CURDIR)/$(ROMFS)/i18n/zh-TW/. $(CURDIR)/$(ROMFS)/i18n/zh-Hant/
-	@$(MAKE) -C $(CURDIR)/TegraExplorer -f $(CURDIR)/TegraExplorer/Makefile
+	@$(MAKE) -C $(CURDIR)/TegraExplorer -f $(CURDIR)/scripts/ryazhenka/tegraexplorer.override.mk
 	@cp $(CURDIR)/TegraExplorer/output/TegraExplorer.bin $(CURDIR)/$(ROMFS)/aio_rcm.bin
 # @$(MAKE) -C $(CURDIR)/aiosu-forwarder -f $(CURDIR)/aiosu-forwarder/Makefile
 	@cp $(CURDIR)/aiosu-forwarder/aiosu-forwarder.nro $(CURDIR)/$(ROMFS)/aiosu-forwarder.nro
