diff --git a/Makefile b/Makefile
index 7ac3ff0..0812ca7 100644
--- a/Makefile
+++ b/Makefile
@@ -20,10 +20,10 @@ SOURCES		:=	source
 RESOURCES	:=	resources
 DATA		:=	data
 INCLUDES	:=	include /lib/borealis/library/include/borealis/extern/nlohmann
-APP_TITLE	:=	All-in-One Switch Updater
-APP_AUTHOR	:=	HamletDuFromage
+APP_TITLE	:=	Ryazhenka Updater
+APP_AUTHOR	:=	Dimasick-git
 APP_VERSION :=	2.23.3
-TARGET		:=	$(notdir $(CURDIR))
+TARGET		:=	ryazhenka-updater
 
 ROMFS				:=	resources
 BOREALIS_PATH		:=	lib/borealis
